packages/ui-library/src/components/toggle-switch/index.ts

```diff
@@ -92,32 +92,32 @@ export class BlrToggleSwitch extends LitElementCustom {
     }
   };
 
-  @state() protected active = false;
+  @state() protected checked = false;
```

Comment:
Perhaps we should rename this to `pressed` so that it doesn't get confused with the checked state. This would also match the token system nomenclature.